.gitignore

```diff
@@ -19,6 +19,7 @@ fofix/core/VideoPlayer/_VideoPlayer.c
 fofix/core/pypitch/_pypitch.cpp
 bin
 build
+dist
```

Comment:
You already ignore it in b06b116. Remove the extra line, then ;).

Reply:
Same change, it just got caught in two different branches. Git is smart enough to merge them.

Reply:
Nope, git will not know that there is another line later that will do that too.

Reply:
When you merge, git sees that nothing has changed and quietly merges the files. There's no conflict here.